tcp.rflx

```diff
@@ -0,0 +1,86 @@
+package TCP is
+
+    -- RFC 793
+
+    type Source_Port is mod 2**16;
+    type Destination_Port is mod 2**16;
+    type Sequence_Number is mod 2**32;
+    type Acknowledgment_Number is mod 2**32;
+    type Data_Offset is range 5 .. 2**4 - 1 with Size => 4;
+    type Reserved is mod 2**3;
```

Comment:
>    Reserved for future use.  __Must be zero.__

Reply:
Reserved is set as zero in commit 4b515169953d78d669625c7fe9e0ebe219cc16e1.